pkg/datapath/config.go

```diff
@@ -54,4 +54,17 @@ type EndpointConfiguration interface {
 	// TODO: Move this detail into the datapath
 	HasIpvlanDataPath() bool
 	ConntrackLocalLocked() bool
+
+	// RequireARPPassthrough returns true if the datapath must implement
+	// ARP passthrough for this endpoint
+	RequireARPPassthrough() bool
+
+	// RequireEgressProg returns true if the endpoint requires an egress
+	// program attached to the InterfaceName() invoking the section
+	// "to-container"
+	RequireEgressProg() bool
+
+	// DisableRouting returns true if the endpoint requires for routing in
+	// BPF to be disabled and delegated to Linux routing
+	DisableRouting() bool
```

Comment:
I would change the naming of this function as it seems the function itself will disable routing.